docs/content/docs/components/cdn/_index.md

```diff
@@ -3,26 +3,35 @@ title: "CDN"
 weight: 3
 ---
 
-<b style="color: red">⚠ Do not activate CDN log processing in production yet. It's in active development.
-Be sure that config flag 'enableLogProcessing' is set to false</b>
-
 ## What's CDN
+CDN is a service dedicated to receive and store CDS's logs. In a near future it will also be able to manage artifacts and cache used by your jobs. 
```

Comment:
```suggestion
CDN is a service dedicated to receive and store CDS's logs. In a near future, it will also be able to manage artifacts and cache used by your jobs. 
```